content/tutorial/tutorial.md

```diff
@@ -12,125 +12,126 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+你不需要有任何 React 的基礎知識就能使用這份學習指南。
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## 在我們開始之前 {#在我們開始之前}
```

Comment:
@chloewlin you don't need to translate anchor link.

Reference: [Universal Style Guide - Heading IDs](https://github.com/reactjs/reactjs.org-translation/blob/master/style-guide.md#heading-ids)

Reply:
@neighborhood999 Oops! Ok I will fix all anchor links now and push it soon.